main.py

```diff
@@ -59,6 +62,11 @@ async def check_scores_task(self):
 
     async def send_wrs(self, scores_json, saved_state):
         channel = self.get_channel(LB_CHANNEL_ID)
+        if not channel:
+            log(f"ERROR: Could not find channel <{LB_CHANNEL_ID}>.")
+            return
+        assert isinstance(channel, discord.TextChannel), "You have set your output channel to a channel that isn't a text channel."
```

Comment:
Instead of having this check twice in two places, make a function to get the channel with checking.

Reply:
Sure thing. I can do that